examples/adc.rs

```diff
@@ -4,9 +4,12 @@ use core::fmt::Write;
 use libtock::adc;
 use libtock::console::Console;
 use libtock::timer;
-use libtock::timer::Duration;
 
 async fn main() {
+    let context = timer::DriverContext::create();
+    let mut driver = context.create_timer_driver().unwrap();
```

Comment:
With results everywhere, `unwrap` should be avoided

Reply:
Right, I'll have to rebase on current master and fix this.

Reply:
Done.